### ISSUE TITLE
SDK no longer works with DPNS-related methods

### ISSUE THREAD

**Issue by thephez** [dashpay/platform#1999]
## Expected Behavior
SDK should be able to register/retrieve names using the `names.*` [methods](https://github.com/dashpay/platform/tree/v1.0-dev/packages/js-dash-sdk/src/SDK/Client/Platform/methods/names).

## Current Behavior
Since https://github.com/dashpay/platform/pull/1984, this error is returned when using `names.register()`:

`Error: JsonSchemaError: Additional properties are not allowed ('dashUniqueIdentityId' was unexpected), path: /records`

Also receiving errors like this when attempting to retrieve a name:
```
InvalidRequestError: where clause on non indexed property error: query must be for valid indexes, valid indexes are: {"identityId": Index { name: "identityId", properties: [IndexProperty { name: "records.identity", ascending: true }], unique: false, null_searchable: false, contested_index: None }, "parentNameAndLabel": Index { name: "parentNameAndLabel", properties: [IndexProperty { name: "normalizedParentDomainName", ascending: true }, IndexProperty { name: "normalizedLabel", ascending: true }], unique: true, null_searchable: true, contested_index: Some(ContestedIndexInformation { field_matches: {"normalizedLabel": Regex(Regex("^[a-zA-Z01-]{3,19}$"))}, resolution: MasternodeVote }) }}
```

## Steps to Reproduce (for bugs)
1. Attempt to create or retrieve a name using the RC package for JS SDK

## Your Environment
* Version used: 4.0.0-rc.1 on testnet

**Comment by thephez** [comment]
Should be resolved by #2001 / #2002

**Comment by riongull** [comment]
`names.register()` with v4.0.0-rc.2 worked for me.
